SpatialGDK/Build/Scripts/ExternalSchemaCodegen.bat

```diff
@@ -0,0 +1,62 @@
+@echo off
+
+IF NOT "%~2"=="" IF "%~3"=="" GOTO START
+ECHO This script requires two parameters:
+ECHO - path to external schema directory
+ECHO - target output folder for generated code

```

Comment:
Can we make it so that path to external directory and codegen output can be (or _is_) specified relative to project's root? And then document it accordingly?

Reply:
The example invocation in the docs is `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\ExternalSchemaCodegen.bat spatial\schema\my_external_schema Game\Source\<your_project_name>\ExternalSchemaCodegen` which is relative to the project root? Can add some documentation in code too if you think?

Reply:
Does that work though, below here the invocation goes 

`%CODEGEN_EXE_PATH% --input-bundle %BUNDLE_CACHE_DIR%\%SCHEMA_BUNDLE_FILE_NAME% --output-dir %2`

So it would treat output dir relative to `cwd` right?

Reply:
Ah yep I see 👍